### DISCUSSION TITLE
Copying a single index across Databases

### DISCUSSION THREAD

**Discussion by tehkyle** [ravendb/ravendb#13663] [Q&A]
In Raven 3.5 there was the ability to copy and paste index definitions across databases:
![image](https://user-images.githubusercontent.com/1372402/154599798-c44bb2c9-7b7c-4402-b90e-114f34d5e6aa.png)
While there is still a "Copy C#" I was unable to find a way to copy and paste the definition itself (previously as json) in the Raven 4.2 database or documentation. Has it been deprecated and is there an alternate approach to this operation?

**Comment by ppekrol** [comment]
Hi,

you are correct. We removed that in 4.x. Best way to copy indexes would be via export/import process. Please be aware that this will copy all of them.

**Reply by tehkyle** [reply]
Is there a technical reason for its removal? Or could I file an issue to return it?

**Reply by ppekrol** [reply]
I've created: https://issues.hibernatingrhinos.com/issue/RavenDB-18066

**Reply by tehkyle** [reply]
Thank you

**Reply by tehkyle** [reply]
Bumping this, any further thoughts on copying a single index across databases?

**Reply by ppekrol** [reply]
hi @tehkyle, we haven't scheduled any work around that, but we are now working on plans for 2024, and I've moved that to 'high interest' list.
